### ISSUE TITLE
Detect dead-letter-queues in smallrye-health

### ISSUE THREAD

**Issue by FelixA** [quarkusio/quarkus#26111]
### Description

smallrye-health automatically detects incoming and outgoing topics as well as their health status and the possibility to verify if the topic exists on the broker.
Unfortunately, this functionality does not yet exist for dead-letter-queues. 
It would be convenient if there was also a detection for the dead-letter-queues.

I would expect, if I either add the property deletion.failure-strategy=dead-letter-queue,
that there was a detection for the topic dead-letter-topic-${brokername}
or if I add the property dead-letter-queue.topic=example-dlq
that there was a a detection for the topic example-dlq

**Comment by Ladicek** [comment]
This is actually a feature request for SmallRye Reactive Messaging, which implements the health checks.

CC @cescoffier @ozangunalp

**Comment by cescoffier** [comment]
Yes, this need to be added into SmallRye Reactive Messaging first.

**Comment by cescoffier** [comment]
I have opened: https://github.com/smallrye/smallrye-reactive-messaging/issues/1818.

**Comment by ozangunalp** [comment]
This is not quite right, because currently for incoming channel client-based health checks, we only check if we can connect to the broker, and do not check the existence of specific topics. We do that for outgoing channels.
